c2corg_api/__init__.py

```diff
@@ -7,19 +8,85 @@
     )
 from c2corg_api.views import cors_policy
 
+from pyramid.security import Allow, Everyone, Authenticated
+
+from c2corg_api.security.roles import validate_token
+
+
+class RootFactory(object):
+    __name__ = 'RootFactory'
+    __acl__ = [
+            (Allow, Everyone, 'public'),
+            (Allow, Authenticated, 'member'),
```

Comment:
'contributor' is maybe more explicit and less confusing compared to the members of the association.
Or simply 'authenticated'.